bitrise.yml

```diff
@@ -1,278 +1,67 @@
+        title: Switch working dir to test / _tmp dir
+        inputs:
+        - path: ./_tmp
+        - is_create_path: true
+    - git::https://github.com/bitrise-steplib/bitrise-step-simple-git-clone.git@master:
+        inputs:
+          - repository_url: https://github.com/bitrise-io/sample-apps-ios-simple-objc-with-uitest.git # Remote or local URL
+          - branch: master
+          - clone_into_dir: .
+    - path::./:
+        inputs:
+        - xcodebuild_test_options: -verbose
+        - export_uitest_artifacts: "true"
+        - verbose: "yes"
 
-  ci-objc_headless_mode_failing_ui_test:
+  sample_retry_after_test_fail:
     envs:
-    - SAMPLE_APP_URL: https://github.com/bitrise-io/sample-apps-ios-simple-objc-with-uitest.git
-    - SAMPLE_APP_BRANCH: failing-ui-test
-    - OUTPUT_TOOL: xcodebuild
     - BITRISE_PROJECT_PATH: ios-simple-objc/ios-simple-objc.xcodeproj
     - BITRISE_SCHEME: ios-simple-objc
-    - SIMULATOR_DEVICE: iPhone 8 Plus
-    - SIMULATOR_OS_VERSION: "latest"
-    - SIMULATOR_PLATFORM: iOS Simulator
-    - HEADLESS_MODE: "yes"
-    - RETRY_ON_FAIL: "no"
-    after_run:
-    - _test
-    - check_exported_artifacts
-
-  tvos_headless_mode:
-    envs:
-    - SAMPLE_APP_URL: https://github.com/bitrise-io/sample-apps-tvos-swift.git
-    - SAMPLE_APP_BRANCH: master
-    - OUTPUT_TOOL: xcodebuild
-    - BITRISE_PROJECT_PATH: NPO Live.xcworkspace
-    - BITRISE_SCHEME: NPO Live
-    - SIMULATOR_DEVICE: Apple TV
-    - SIMULATOR_OS_VERSION: "latest"
-    - SIMULATOR_PLATFORM: tvOS Simulator
-    - HEADLESS_MODE: "yes"
-    - RETRY_ON_FAIL: "no"
-    after_run:
-    - _test
-
-  ci-parallel_headless_mode:
-    envs:
-    - SAMPLE_APP_URL: https://github.com/bitrise-io/sample-swift-project-with-parallel-ui-test.git
-    - SAMPLE_APP_BRANCH: master
-    - OUTPUT_TOOL: xcodebuild
-    - BITRISE_PROJECT_PATH: BullsEye.xcodeproj
-    - BITRISE_SCHEME: BullsEye
-    - SIMULATOR_DEVICE: iPhone 8 Plus
-    - SIMULATOR_OS_VERSION: "latest"
-    - SIMULATOR_PLATFORM: iOS Simulator
-    - HEADLESS_MODE: "yes"
-    - RETRY_ON_FAIL: "no"
-    after_run:
-    - _test
-
-  _test:
     steps:
     - script:
         inputs:
         - content: |-
             #!/bin/env bash
-            set -e
-            set -v
+            set -ex
             rm -rf ./_tmp
     - change-workdir:
         title: Switch working dir to test / _tmp dir
-        run_if: true
         inputs:
         - path: ./_tmp
         - is_create_path: true
-    - script:
-        inputs:
-        - content: |-
-            #!/bin/env bash
-            echo BITRISE_SOURCE_DIR: ${BITRISE_SOURCE_DIR}
-            echo ORIG_BITRISE_SOURCE_DIR: ${ORIG_BITRISE_SOURCE_DIR}
-            echo working directory: $(pwd)
     - git::https://github.com/bitrise-steplib/bitrise-step-simple-git-clone.git@master:
         inputs:
-          - repository_url: $SAMPLE_APP_URL
+          - repository_url: https://github.com/bitrise-io/sample-apps-ios-simple-objc-with-uitest.git
```

Comment:
I think it is better to move this config into env var, repository URL and branch might need to be changed to test the current changes during development

Reply:
Extracted